scr/main/java/com/globant/services/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package com.globant.services;
+
+import com.globant.models.Product;
+import com.globant.models.ShoppingCart;
+import com.globant.repositories.CartRepository;
+
+/**
+ * This class implements the ShoppingCartService interface
+ * 
+ * @author Yael Salim
+ *
+ */
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+
+	CartRepository shoppingCarts = new CartRepository();
+
+	/**
+	 * Creates a new shopping cart for a user

```

Comment:
You don't need to replicate the javadoc in the implementation, if you add this to the interface it's enough